packages/now-next/package.json

```diff
@@ -19,18 +19,20 @@
   ],
   "devDependencies": {
     "@types/aws-lambda": "8.10.19",
+    "@types/buffer-crc32": "0.2.0",
     "@types/fs-extra": "8.0.0",
-    "@types/jszip": "3.1.6",
     "@types/next-server": "8.0.0",
     "@types/resolve-from": "5.0.1",
     "@types/semver": "6.0.0",
+    "@types/yazl": "2.4.1",
     "@zeit/node-file-trace": "0.2.13",
     "async-sema": "3.0.1",
+    "buffer-crc32": "0.2.13",
     "fs-extra": "7.0.0",
     "get-port": "5.0.0",
-    "jszip": "3.2.2",
     "resolve-from": "5.0.0",
     "semver": "6.1.1",
-    "typescript": "3.5.2"
+    "typescript": "3.5.2",
+    "yazl": "https://github.com/ijjk/yazl#70949c55b482647669ce37023017b1514c42b33c"
```

Comment:
Is there an upstream PR for the relevant change in your fork? If not, perhaps we should publish under `@zeit` scope on the registry.

Reply:
We're sending a PR, no reason to fork in the intermediary time IMO.

Reply:
PR here https://github.com/thejoshwolfe/yazl/pull/55